### PR TITLE
Fix incompatible versions of `black` and `click`

### PR DESCRIPTION
**Pull request by mayaCostantini**
## Related Issues and Dependencies

Related to https://github.com/thoth-station/kebechet/pull/1042

## This introduces a breaking change

- No

## This should yield a new module release

- No

### CODE REVIEW

File: prescriptions/bl_/black/click8x10x0_black22x1x0.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  steps:
+  - name: Click8x10x0Black22x1x0Step
+    type: step.Group
+    should_include:
+      adviser_pipeline: true
+    match:
+    - group:
+      - package_version:
+          name: click
+          version: <8.10.0
```
Comment (harshad16) [inline prescriptions/bl_/black/click8x10x0_black22x1x0.yaml:11]:
```suggestion
          version: <8.1.0
```

**Review by harshad16 — CHANGES_REQUESTED** [review]
nice one :100: 
just one comment: 
https://pypi.org/project/click/#history

File: prescriptions/bl_/black/click8x10x0_black22x1x0.yaml (hunk 1 above)
Comment (mayaCostantini) [reply prescriptions/bl_/black/click8x10x0_black22x1x0.yaml:11]: Thanks! :100: Fixed in https://github.com/thoth-station/prescriptions/pull/23484/commits/8335af62bcef18968046f4064601f23c360f58f3

**Review by harshad16 — APPROVED** [review]
/lgtm
thanks :100:

**Pull request merged**